src/performance_test_fixture.cpp

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "performance_test_fixture/performance_test_fixture.hpp"
+
+#include <algorithm>
+#include <cstdlib>
+#include <vector>
+
+#ifdef _WIN32
+#pragma warning(disable : 4996)
+#endif
+
+performance_test_fixture::PerformanceTest::PerformanceTest()
+: suppress_memory_tools_logging(true)
+{
+  const char * performance_test_fixture_enable_trace = getenv(
```

Comment:
@cottsay should we be using `rcutils_get_env` instead?

Reply:
I think it is a goal to use this package to do performance testing in `rcutils`, so we can't take that dependency.